### ISSUE TITLE
Support multiple versions of Blaze documentation on our site

### ISSUE THREAD

**Issue by mitar** [meteor/blaze#77]
So we should make the site/docs describe current version of Blaze. But as we will be adding more and more features to Blaze it would be great if users could navigate to old versions of documentation as well.

**Comment by alexandesigner** [comment]
Currently on the site it is possible to navigate by version [See](https://github.com/meteor/blaze/blob/master/site/_config.yml#L18) (as it is done in the Meteor Docs, as well as the search too, just need to better understand how exactly work this versioning on site..

I speak with regard to content, how to organize it?

**Comment by mitar** [comment]
@stubailo, @tmeasday: How is that versioning working?

I think we should track it based on Blaze package version. Other packages are mostly dependencies. And we can also in the worst case bump Blaze package version if some other package has a large change which has to be documented well.

**Comment by tmeasday** [comment]
It works like:

1. Create a branch called `versionX`, this will automatically deploy to http://blazejs.org/vX (I think?)
2. Add values to the `versions` field of `_config.yml` ([here](https://github.com/meteor/blaze/blob/master/site/_config.yml#L18)). When you have more than one the version picker will appear.

**Comment by mitar** [comment]
Oh, hm. Currently on every `master` branch change only, the site is deployed to `gh-pages`. Not sure about any branches.

I do not find any documentation about these `versions` config?

**Comment by tmeasday** [comment]
Oh, right i forgot you weren't deploying to s3.

Well it'll work if you switch to using that. Otherwise I guess you can try and figure out a similar solution.

**Comment by mitar** [comment]
Hm, but even with S3, where you have code which is deploying those branches? And where you configure that logic? GitHub hooks? CircleCI?

**Comment by tmeasday** [comment]
https://github.com/meteor/hexo-s3-deploy

**Comment by mitar** [comment]
OK, we are now deploying to S3 to a bucket hosted by MDG (`blazejs.org.s3-website-us-east-1.amazonaws.com`). I had to configure CNAME for top-level domain (`blazejs.org`) because Namecheap does not support ALIAS or ANAME DNS entries. This is bad practice and could prevent e-mail to work for our domain, but because we are not really using it, I think it is good enough for now.

I removed GitHub pages.

This allows us now to use the same script and approach Meteor is using for their versions. That is, every release we want to have documentation for, we create a `version-*` branch.
